README.md

```diff
@@ -42,3 +42,4 @@ This committee meets at 21:00 UTC on the second Wednesday of every month. All co
 - Tony Qu - @tonyqus
 - SeanKilleen - Github [@SeanKilleen](https://github.com/SeanKilleen), Twitter [@sjkilleen](https://twitter.com/sjkilleen)
 - Aaron LaBeau - @biozal 
+- Alper EBICOGLU - @ebicoglu 
```

Comment:
Why ALL CAPS for your last name?

Reply:
See https://github.com/dotnet-foundation/wg-maintainers/pull/47